other/perun-propagate/debian/changelog

```diff
@@ -1,3 +1,12 @@
+perun-propagate (3.0.4) stable; urgency=low
+
+  * add perun_propagate while instalation to initialization after system
+    restart for deb and rpm pacakges
+  * for deb packages use postinst
+  * for rpm packages use %post, %preun and %postun
```

Comment:
Is it necessary to write differences between rpm and deb packages? I know we are using one changelog for both, but it will look strange if someone will read it in the final package changelog.